src/main/java/uk/nhs/digital/nhsconnect/nhais/repository/SequenceRepository.java

```diff
@@ -16,9 +16,9 @@
 @Slf4j
 @Repository
 public class SequenceRepository {
+    public final static Long MAX_SEQUENCE_NUMBER = 100000000L;
```

Comment:
as mentioned before, recommend this stay private since its related to the implementation of SequenceRepository. The name is also no great since its not the "maximum" but rather a modulus used to enforce the maximum.